### PR TITLE
Bump version to 7.3.0

### PR DESCRIPTION
**Pull request by mikkka**
Bump open api generator version to 7.3.0

@wing328 please have a look

### CODE REVIEW

File: .java-version [hunk 1]
```diff
@@ -1 +1 @@
-1.8
+17
```
Comment (wing328) [inline .java-version:1]: should it be 11 as that's the mininal JDK version supported at the moment.

**Comment by wing328** [comment]
The CI tests failed. Please take a look when you've time.

File: .java-version (hunk 1 above)
Comment (mikkka) [reply .java-version]: fixed 11

File: src/main/scala/org/openapitools/generator/sbt/plugin/OpenApiGeneratorPlugin.scala [hunk 2]
```diff
@@ -45,7 +45,7 @@ object OpenApiGeneratorPlugin extends sbt.AutoPlugin
   val OpenApiCodegen = config("openApiCodegen")
 
   override def globalSettings: Seq[Def.Setting[_]] = Seq(
-    aggregate in openApiGenerators := false
+    openApiGenerators / aggregate := false
```
Comment (wing328) [inline src/main/scala/org/openapitools/generator/sbt/plugin/OpenApiGeneratorPlugin.scala:48]: can you please elaborate on this change?

looks like this caused the CI tests to fail: https://github.com/OpenAPITools/sbt-openapi-generator/actions/runs/7920353292/job/21636830291?pr=48

**Comment by wing328** [comment]
can you please update https://github.com/OpenAPITools/sbt-openapi-generator/actions/runs/7927084946/workflow?pr=48#L29 to jdk11 as well?

**Comment by mikkka** [comment]
Hi @wing328 , do you think that it's time for a new release?

**Comment by mikkka** [comment]
Hi @wing328 , would you kindly release new version?

**Comment by wing328** [comment]
done via https://github.com/OpenAPITools/sbt-openapi-generator/releases/tag/v7.3.0

**Comment by wing328** [comment]
the publish failed since bintray has been unset: https://github.com/OpenAPITools/sbt-openapi-generator/actions/runs/7961025131/job/21731316245

i'll try to see if i can manually publish it this weekend

**Comment by wing328** [comment]
@mikkka does the staging looks good to you?

https://oss.sonatype.org/service/local/repositories/orgopenapitools-1346/content/org/openapitools/sbt-openapi-generator_2.12_1.0/7.3.0+0-4ae9bf0a+20240226-0016/sbt-openapi-generator-7.3.0+0-4ae9bf0a+20240226-0016.jar

**Comment by wing328** [comment]
fyi. filed https://github.com/OpenAPITools/sbt-openapi-generator/pull/49 to switch to sonatype for publishing

**Comment by mikkka** [comment]
> @mikkka does the staging looks good to you?
> 
> https://oss.sonatype.org/service/local/repositories/orgopenapitools-1346/content/org/openapitools/sbt-openapi-generator_2.12_1.0/7.3.0+0-4ae9bf0a+20240226-0016/sbt-openapi-generator-7.3.0+0-4ae9bf0a+20240226-0016.jar

Jar looks good. Wait for it being placed in nexus, so it can be included in project via `addSbtPlugin("org.openapitools" % "sbt-openapi-generator" % "7.3.0")`

**Comment by wing328** [comment]
just did a released of 7.3.0. can you please give it a try?

**Comment by wing328** [comment]
thanks for checking. i'll take another look tomorrow

**Comment by wing328** [comment]
just published to https://repo1.maven.org/maven2/org/openapitools/sbt-openapi-generator_2.12_1.0/7.3.0/ by skipping the version plugin

can you please give it another try?

if it works, i'll cut another release for openapi generator v7.4.0

**Comment by wing328** [comment]
good.

can you please give it a try to see if the sbt plugin works for you as expected?

**Comment by mikkka** [comment]
Hi,
Seems to work fine.

вт, 26 мар. 2024 г. в 18:41, William Cheng ***@***.***>:

**Comment by wing328** [comment]
thanks for testing it

just published v7.4.0: https://central.sonatype.com/artifact/org.openapitools/sbt-openapi-generator/7.4.0

**Comment by pan3793** [comment]
why drop support Java 8

**Comment by wing328** [comment]
it's reached EOL a while ago.

if you can't use JDK11+ in your environment, please use docker instead, e.g. https://hub.docker.com/r/openapitools/openapi-generator-cli

**Pull request merged**